File: yandex_code_life_balance/3. Села батарейка/solution.cs
using System;

class Program
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        string[] input = Console.ReadLine().Split();
        int totalConsumption = 0;
        for (int i = 0; i < n; i++)
            totalConsumption += int.Parse(input[i]);

        Console.WriteLine(100 / totalConsumption);
    }
}

File: yandex_code_life_balance/3. Села батарейка/solution.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main(args: Array<String>) {
	val reader = BufferedReader(InputStreamReader(System.`in`))
	val writer = BufferedWriter(OutputStreamWriter(System.out))
	
	val n = reader.readLine().toInt()
    val consumptions = reader.readLine().split(" ").map { it.toInt() }
    val totalConsumption = consumptions.sum()
    val fullHours = 100 / totalConsumption
	writer.write(fullHours.toString())

	reader.close()
	writer.close()
}